Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionCrowdStrikeFalconHost.yaml

```diff
@@ -0,0 +1,282 @@
+              DstPortNumber = remotePort;
+      let outboundfirewalldata = unparsedfirewalldata
+          | where connectionDirection == 2
+          | extend NetworkDirection = "Outbound"
+          | project-rename
+              SrcIpAddr = remoteAddress,
+              SrcPortNumber = remotePort,
+              DstIpAddr = localAddress,
+              DstPortNumber = localPort;
+      let parsedfirewalldata = union inboundfirewalldata, outboundfirewalldata
+          | extend deviceIp = iff(hostName matches regex "(([0-9]{1,3})\\.([0-9]{1,3})\\.([0-9]{1,3})\\.(([0-9]{1,3})))", hostName, "")
+          | extend 
+              hostName = iff(isempty(deviceIp), hostName, ""),
+              AdditionalFields = bag_pack(
+                      "networkProfile", networkProfile,
+                      "ruleDescription", ruleDescription,
+                      "ruleGroupName", ruleGroupName,
+                      "cmdLine", cmdLine
+                  ),
+              NetworkIcmpCode = icmpCode
+          | invoke _ASIM_ResolveDvcFQDN('hostName')
+          | invoke _ASIM_ResolveNetworkProtocol('protocol')
+          | extend NetworkIcmpType = _ASIM_LookupICMPType('icmpType')
+          | lookup EventFieldsLookup on ruleAction
+          | project-rename
+              DvcId = deviceId,
+              DvcIpAddr = deviceIp,
+              EventOriginalSubType = eventType,
+              NetworkRuleName = ruleName
+          | extend
+              Rule = NetworkRuleName,
+              Dvc = coalesce(DvcId, DvcHostname, DvcIpAddr);
+      let networkaccessdata = alldata
+          | where DeviceEventClassID has "Network Access In A Detection Summary Event"
+          | parse-kv AdditionalExtensions as (CSMTRPatternDisposition: string) with (pair_delimiter=";", kv_delimiter="=")
+          | lookup ActionLookup on EventOutcome
+          | invoke _ASIM_ResolveSrcFQDN('DestinationHostName')
+          | extend
+              EventStartTime = todatetime(DeviceCustomDate1),
+              DstIpAddr = coalesce(DestinationIP, DeviceCustomIPv6Address3),
+              SrcIpAddr = coalesce(SourceIP, DeviceCustomIPv6Address2),
+              EventCount = int(1),
+              EventType = "EndpointNetworkSession",
+              SrcDomain = coalesce(DestinationNTDomain, SrcDomain),
+              EventOriginalResultDetails = CSMTRPatternDisposition,
+              SrcProcessId = tostring(FieldDeviceCustomNumber2),
+              SrcDomainType = iff(isnotempty(DestinationNTDomain), "Windows", SrcDomainType),
+              AdditionalFields = bag_pack("CSMTRPatternDisposition", CSMTRPatternDisposition)
+          | project-rename
+              DvcId = ExtID,
+              DstPortNumber = DestinationPort,
+              SrcPortNumber = SourcePort,
+              SrcMacAddr = SourceMACAddress,
+              SrcUsername = DestinationUserName,
+              SrcProcessName = FileName
+          | extend
+              DvcHostname = SrcHostname,
```

Comment:
Did you validate? I don't think it will always be same.  Only when direction is outbound Src would be same as Dvc.

Reply:
Removed this mapping as we don't have direction information available in logs.